handbook/engineering/distribution/index.md

```diff
@@ -132,3 +132,13 @@ Examples:
 - Measure and sustain 99.99% uptime and reliability of Sourcegraph Cloud.
 - Enforce access controls and network security policies for our Cloud resources.
 - Make it easy for customers to share relevant data when submitting bug reports.
+
+
+### Other processes
+#### Backlog grooming
+
+The distribution team runs a backloog grooming session every two weeks to ensure that backlog is correctly priorotized for the upcoming sprint. 
```

Comment:
bäckög

Reply:
I wrote it 5 times, I was bound to screw it up at least once.